mongo/document/validation_error.go

```diff
@@ -0,0 +1,43 @@
+package document
+
+import "bytes"
+
+type ValidationError struct {
```

Comment:
I would prefer calling it `ValidationErrors`

Reply:
:heavy_check_mark: